src/panel/details.tsx

```diff
@@ -36,47 +36,47 @@ interface DetailsProps { result: Result, height: IObservableValue<number> }
+                : renderMessageWithEmbeddedLinks(result, vscode.postMessage, desc.text);
         };
 
         const {result, height} = this.props;
         const helpUri = result?._rule?.helpUri;
         const renderLocation = (location: Location) => {
-			const { message, uri, region } = parseLocation(result, location)
+			const { message, uri, region } = parseLocation(result, location);
 			return <>
 				<div className="ellipsis">{message ?? '—'}</div>
 				<div className="svSecondary">{uri?.file ?? '—'}</div>
 				<div className="svLineNum">{region?.startLine}:1</div>
-			</>
-		}
+			</>;
+		};
 		const renderStack = (stackFrame: StackFrame) => {
-			const location = stackFrame.location
-			const logicalLocation = stackFrame.location?.logicalLocations[0]
-			const { message, uri, region } = parseLocation(result, location)
-			const text = `${message ?? ''} ${logicalLocation?.fullyQualifiedName ?? ''}`
+			const location = stackFrame.location;
+			const logicalLocation = stackFrame.location?.logicalLocations[0];
+			const { message, uri, region } = parseLocation(result, location);
+			const text = `${message ?? ''} ${logicalLocation?.fullyQualifiedName ?? ''}`;
 			return <>
 				<div className="ellipsis">{text ?? '—'}</div>
 				<div className="svSecondary">{uri?.file ?? '—'}</div>
 				<div className="svLineNum">{region?.startLine}:1</div>
-			</>
-		}
+			</>;
+		};
         return <div className="svDetailsPane" style={{ height: height.get() }}>
             {result && <TabPanel selection={this.selectedTab}>
                 <Tab name="Info">
                     <div className="svDetailsBody svDetailsInfo">
                         <div className="svDetailsMessage">
                             {result._markdown
                                 ? <ReactMarkdown className="svMarkDown" source={result._markdown} escapeHtml={false} />
-                                : renderMessageWithEmbeddedLinks(result, vscode.postMessage)}</div>
+                                : renderMessageWithEmbeddedLinks(result, vscode.postMessage, result._message)}</div>
                         <div className="svDetailsGrid">
                             <span>Rule Id</span>			{helpUri ? <a href={helpUri} target="_blank" rel="noopener noreferrer">{result.ruleId}</a> : <span>{result.ruleId}</span>}
                             <span>Rule Name</span>			<span>{result._rule?.name ?? '—'}</span>
-                            <span>Rule Desc Short</span>	<span>{renderRuleDesc(result._rule?.shortDescription)}</span>
-                            <span>Rule Desc Full</span>		<span>{renderRuleDesc(result._rule?.fullDescription)}</span>
+                            <span>Rule Desc Short</span>	<span><div>{renderRuleDesc(result, result._rule?.shortDescription)}</div></span>
+                            <span>Rule Desc Full</span>		<span><div>{renderRuleDesc(result, result._rule?.fullDescription)}</div></span>
```

Comment:
Reason for new `div`?

Reply:
Was having this issue: https://stackoverflow.com/questions/52529682/how-do-you-make-links-href-in-css-grid-appear-inline-versus-block

Reply:
Thanks. Fixed